package.json

```diff
@@ -34,6 +34,7 @@
     "ignore-styles": "^5.0.1",
     "jsdom": "^11.5.1",
     "lerna": "^2.2.0",
+    "node-abi": "^2.18.0",
```

Comment:
Do you understand why yarn wanted to defined `node-abi` at the root?
_Note_: I confirmed that it is license compatible following our _clearly defined_ procedure.

```
gitpod /workspace/theia $ yarn why node-abi
yarn why v1.22.4
[1/4] Why do we have the module "node-abi"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "node-abi@2.21.0"
info Has been hoisted to "node-abi"
info Reasons this module exists
   - "workspace-aggregator-75ad6f74-e941-4ed8-8275-2ec786df70d0" depends on it
   - Specified in "devDependencies"
   - Hoisted from "_project_#node-abi"
   - Hoisted from "_project_#@theia#application-manager#electron-rebuild#node-abi"
   - Hoisted from "_project_#@theia#core#drivelist#prebuild-install#node-abi"
info Disk size without dependencies: "80KB"
info Disk size with unique dependencies: "164KB"
info Disk size with transitive dependencies: "164KB"
info Number of shared dependencies: 1
Done in 1.09s.
```

Reply:
It added it the first time, but not when doing it a second time. I removed it for now.